css/css-fonts/variations/font-opentype-collections.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>

```

Comment:
This is not variable fonts test, so you may want to move it outside variations folder. But otherwise test is useful and should also be expanded to test access to individual fonts inside collection.